packages/experiments/src/components/SelectedItemsList/BaseSelectedItemsList.tsx

```diff
@@ -69,24 +74,24 @@ export class BaseSelectedItemsList<T, P extends IBaseSelectedItemsListProps<T>>
 
   @autobind
   public onCopy(ev: React.ClipboardEvent<HTMLElement>): void {
-    if (this.props.onCopyItems && this.props.selection.getSelectedCount() > 0) {
-      let selectedItems: T[] = this.props.selection.getSelection() as T[];
+    if (this.props.onCopyItems && this.selection.getSelectedCount() > 0) {
+      let selectedItems: T[] = this.selection.getSelection() as T[];
       this.copyItems(selectedItems);
     }
   }
 
   public unselectAll(): void {
-    this.props.selection.setAllSelected(false);
+    this.selection.setAllSelected(false);
   }
 
   public componentWillUpdate(newProps: P, newState: IBaseSelectedItemsListState): void {
```

Comment:
Since props can also contain selection, you also need to override `this.selection` if they pass in a new selection.